logstash-forwarder.go

```diff
@@ -217,7 +209,7 @@ func emit(msgfmt string, args ...interface{}) {
 	if options.quiet {
 		return
 	}
-	infolog.Printf(msgfmt, args...)
```

Comment:
"info" log didn't make sense to me. the emit() function already takes care of the "info-level" style logs by doing a `options.quiet` check.